### PR TITLE
Adding a new user to ocvn. 

### PR DESCRIPTION
**Pull request by onurd86**
We need to update the database initialization script and we need to add the new user to EmbeddedController and ExternalController. For adding the password into the database, the password is encoded using StandardPasswordEncoder in Spring. This encoder uses SHA-256 hashing with 1024 iterations and a random 8-byte random salt value.

### CODE REVIEW

File: jdk_8_maven/em/embedded/rest/ocvn/src/main/java/em/embedded/org/devgateway/ocvn/EmbeddedEvoMasterController.java [hunk 1]
```diff
@@ -146,7 +146,12 @@ public void resetStateOfSUT() {
 
     @Override
     public List<AuthenticationDto> getInfoForAuthentication() {
-        return Arrays.asList(AuthUtils.getForDefaultSpringFormLogin("ADMIN", "admin", "admin"));
+
+        // Adding a new user called user1 with password "password". User details are stored in the database.
```
Comment (arcuri82) [inline jdk_8_maven/em/embedded/rest/ocvn/src/main/java/em/embedded/org/devgateway/ocvn/EmbeddedEvoMasterController.java:150]: this comment is something would make sense on Git, not here.

File: jdk_8_maven/em/embedded/rest/ocvn/src/main/resources/init_db.sql [hunk 2]
```diff
@@ -34,6 +34,35 @@ VALUES
...
+     NULL,
+     'user1',
+     NULL
+);
+
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,2);
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,1);
 INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(1,3);
+
+INSERT INTO PERSON_ROLES(PERSON_ID,ROLES_ID)VALUES(2,1);
```
Comment (arcuri82) [inline jdk_8_maven/em/embedded/rest/ocvn/src/main/resources/init_db.sql:68]: shouldn't `jdk_8_maven/em/external/rest/ocvn/src/main/resources/init_db.sql` be modified as well?

File: jdk_8_maven/em/external/rest/ocvn/src/main/java/em/external/org/devgateway/ocvn/ExternalEvoMasterController.java [hunk 3]
```diff
@@ -239,7 +239,12 @@ public SutInfoDto.OutputFormat getPreferredOutputFormat() {
 
     @Override
     public List<AuthenticationDto> getInfoForAuthentication() {
-        return Arrays.asList(AuthUtils.getForDefaultSpringFormLogin("ADMIN", "admin", "admin"));
+
+        // Adding a new user called user1 with password "password". User details are stored in the database.
```
Comment (arcuri82) [inline jdk_8_maven/em/external/rest/ocvn/src/main/java/em/external/org/devgateway/ocvn/ExternalEvoMasterController.java:243]: see previous comments. `Adding a new user called user1 with password "password"` can be removed. regarding database, can mention `init_db.sql` script in the comment here

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: jdk_8_maven/em/embedded/rest/ocvn/src/main/java/em/embedded/org/devgateway/ocvn/EmbeddedEvoMasterController.java (hunk 1 above)
Comment (onurd86) [reply jdk_8_maven/em/embedded/rest/ocvn/src/main/java/em/embedded/org/devgateway/ocvn/EmbeddedEvoMasterController.java]: Hi Andrea, thanks for your comment. I fixed that.

File: jdk_8_maven/em/embedded/rest/ocvn/src/main/resources/init_db.sql (hunk 2 above)
Comment (onurd86) [reply jdk_8_maven/em/embedded/rest/ocvn/src/main/resources/init_db.sql:69]: Hi Andrea, thanks for your comment. I added those to both init_db.sql scripts (for embedded and external).

File: jdk_8_maven/em/external/rest/ocvn/src/main/java/em/external/org/devgateway/ocvn/ExternalEvoMasterController.java (hunk 3 above)
Comment (onurd86) [reply jdk_8_maven/em/external/rest/ocvn/src/main/java/em/external/org/devgateway/ocvn/ExternalEvoMasterController.java]: Hi Andrea, thanks for your comment. I fixed that.

**Review by arcuri82 — APPROVED** [review]

**Pull request merged**